chianft/util/mint.py

```diff
@@ -227,6 +230,7 @@ async def add_fee_to_spend(
             coins=[fee_coin],
             fee=uint64(total_fee),
         )
+        assert isinstance(fee_tx.spend_bundle, SpendBundle)
```

Comment:
What else would the spend bundle be?  If it is just `Optional` then `is not None` would be more clear.  Also, could we instead raise an exception that explains how this might happen?  When asserts are used for something other than a test passing condition, they should generally provide a message as well.

Reply:
have changed all the `isinstance`s to `is not None`